src/hooks/useJumperTracking.ts

```diff
@@ -0,0 +1,122 @@
+'use client';
+import { usePathname } from 'next/navigation';
+
+import {
+  JUMPER_ANALYTICS_EVENT,
+  JUMPER_ANALYTICS_TRANSACTION,
+} from 'src/const/abi/jumperApiUrls';
+import { useFingerprint } from './useFingerprint';
+interface JumperDataTrackEventProps {
+  category: string;
+  action: string;
+  label: string;
+  value?: number;
+  data?: { [key: string]: string | number | boolean };
+  isConnected?: boolean;
+  walletAddress?: string;
+  browserFingerprint?: string;
+  isMobile: boolean;
+  sessionId?: string;
+}
```

Comment:
correct interface:

interface JumperDataTrackEventProps {
  category: string;
  action: string;
  label: string;
  url: string;
  value: number;
  data?: { [key: string]: string | number | boolean };
  isConnected: boolean;
  ip?: string;
  walletAddress?: string;
  browserFingerprint: string;
  location: string;
  sessionId: string;
  browserUserAgent?: string;
  isMobile: boolean;
  walletAddress?: string;
}

Reply:
I applied it